binary/src/Cardano/Binary/Deserialize.hs

```diff
@@ -81,6 +82,17 @@ decodeFullDecoder lbl decoder bs0 = case deserialiseDecoder decoder bs0 of
     else Left $ DecoderErrorLeftover lbl leftover
   Left (e, _) -> Left $ DecoderErrorDeserialiseFailure lbl e
 
+decodeFullDecoder'
```

Comment:
Does it pain you to have to propagate the ticks, or do you like it when the meaning is about strictness?